main.tf

```diff
@@ -167,7 +167,8 @@ module "ecs_alb_service_task" {
   subnet_ids                         = var.ecs_private_subnet_ids
   container_port                     = var.container_port
   nlb_container_port                 = var.nlb_container_port
-  docker_volumes                     = var.volumes
+  docker_volumes                     = [for volume in var.volumes : volume if length(volume.docker_volume_configuration) > 0]
```

Comment:
These conditions might be better in the ecs alb service task module instead of here

https://github.com/cloudposse/terraform-aws-ecs-alb-service-task

Reply:
@nitrocode Wouldn't that require the ecs alb service task module to be refactored to accept a generic `volumes` variable?

Reply:
Hmm i don't imagine there would be much of a refactoring. I'll defer to the @cloudposse/contributors if they have an opinion on it

Reply:
Happy to move this to ecs alb service task if needed. Just let me know what needs to be done to get this fix merged please